src/prompts/password.rs

```diff
@@ -1,3 +1,5 @@
+use std::borrow::Cow;
+
 use crate::{
```

Comment:
comment to do something with the current examples so that both confirmation and w/o confirmation are present there for anyone interested